src/dhanshri/Coding_exam_1/NumTargetIndices.java

```diff
@@ -0,0 +1,34 @@
+package dhanshri.Coding_exam_1;
+
+public class NumTargetIndices {
+	
+    int[] indicesNumtarget(int[] input, int tNum) {
+    	 // int[] nextNum = new int[2];
+    	for (int index=0;index<input.length;index++) {
+    		
+    		for (int index1=1;index<input.length;index++)
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files